pkg/config/config.go

```diff
@@ -194,6 +197,58 @@ func (c Config) String() string {
 	return string(b)
 }
 
+// GetJiraFieldKey returns the jira key associated to a field.
```

Comment:
Comment is different vs actual function signature -- GetJiraFieldKey only returns error or nil.